controllers/compassmanager_controller.go

```diff
@@ -413,7 +413,7 @@ func createCompassRuntimeLabels(kymaLabels map[string]string) map[string]interfa
 	runtimeLabels["broker_instance_id"] = kymaLabels[LabelBrokerInstanceID]
 	runtimeLabels["gardenerClusterName"] = kymaLabels[LabelShootName]
 	runtimeLabels["subaccount_id"] = kymaLabels[LabelSubaccountID]
-	runtimeLabels["global_account_id"] = kymaLabels[LabelGlobalAccountID]
+	runtimeLabels["global_subaccount_id"] = kymaLabels[LabelGlobalAccountID]
```

Comment:
You're still taking the `LabelGlobalAccountID`, should it be a `LabelSubaccountID` instead?

Reply:
don't know the logic in details and the variable naming sounds at least suspicious to me

Reply:
I applied the changes